source/Account.h

```diff
@@ -21,75 +21,181 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include <cstdint>
 #include <map>
 #include <string>
+#include <tuple>
 #include <vector>
 
 class DataNode;
 class DataWriter;
 
+// A utility stucture to return multpile values needed during processing
+// inside the Step function and its subprocesses.
+struct Receipt {
+	bool paidInFull = true;
+	int64_t creditsPaid = 0;
+};
 
 
 // Class representing all your assets and liabilities and tracking their change
 // over time.
 class Account {
 public:
-	// Load or save account data.
+	// Load account information from a data file (saved game or starting conditions).
 	void Load(const DataNode &node, bool clearFirst);
+
+	// Write account information to a saved game file.
 	void Save(DataWriter &out) const;
 
-	// Get or change the player's credits.
+	// Get much the player currently has in the bank.
```

Comment:
```suggestion
	// Get how much credits the player currently has in the bank.
```

Reply:
```suggestion
	// Get how many credits the player currently has in the bank.
```
Did you mean like this?